.github/ISSUE_TEMPLATE/new_project.md

```diff
@@ -0,0 +1,39 @@
+---
+name: New Helix Customer Project
+about: Checklist for the creation of a new customer project
+title: ''
+assignees: ''
+
+---
+
+## Start a new Helix customer project
+
+### Set up the site
+
+- [ ] 1. Create a new GitHub repository, either under [Helix Sites](https://github.com/organizations/hlxsites/repositories/new) or the customer's own GitHub organization
+- [ ] 2. Create a content folder, either in the dedicated SharePoint site for [Helix Projects](https://adobe.sharepoint.com/sites/HelixProjects/Shared%20Documents/Forms/AllItems.aspx?id=%2Fsites%2FHelixProjects%2FShared%20Documents%2Fsites)the customer's own Google Drive or SharePoint
+- [ ] 3. Share the folder with `helix@adobe.com` (Helix Bot) and copy the URL
```

Comment:
this is a bit tricky.... especially for sharepoint sites. it should not be the sharelink, but a relatively clean url, like:

https://adobe.sharepoint.com/sites/HelixProjects/Shared%20Documents/sites/westjet

Reply:
Ok, but how can we explain this so most people get it right the first time around? Or should we rather improve the system to be more resilient towards exotic sharepoint URLs?

Reply:
we could either try to sanitize the urls when computing the contentbus id, or
reject the fstab if it's not clean and create a PR/issue to address the problem.